### PR TITLE
Fixes #180

### LINKED ISSUE #180: Documentation for Seq.tail is missing
The latest FSharp libraries added `Seq.tail` in the `Collections.Seq` module. Source code is [here](https://github.com/fsharp/fsharp/blob/37a100b7caafde0f4df5a1924c9f65f4a18277a8/src/fsharp/FSharp.Core/seq.fs#L1794) However, the documentation is still missing for it.

### PR DESCRIPTION
**Pull request by liboz**
Not sure exactly what's needed for the docs. Here's my initial attempt.

### CODE REVIEW

File: docs/conceptual/seq.tail['t]-function-[fsharp].md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+|----|----|
+|[ArgumentException](https://msdn.microsoft.com/library/system.argumentexception.aspx)|Thrown when the input sequence is empty.|
+|[ArgumentNullException](https://msdn.microsoft.com/library/system.argumentnullexception.aspx)|Thrown when the input sequence is null.|
+
+## Return Value
+
+The result sequence.
+
+## Remarks
+This function is named **Tail** in compiled assemblies. If you are accessing the function from a language other than F#, or through reflection, use this name.
```
Comment (dend) [inline docs/conceptual/seq.tail['t]-function-[fsharp].md]: Could you please use a backtick for `Tail` instead of bold?

**Comment by dend** [comment]
@cartermp @KevinRansom can you please validate the accuracy? Structure-wise this looks good.

File: docs/conceptual/snippets/fssequences/snippet204.fs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+let seq1 = seq {1..10}
+let seq2 = Seq.empty
+
+//The following line prints seq [2; 3; 4; 5; ...]
+printfn "%A" (Seq.tail seq1)
+//The following line print this error message:
```
Comment (cartermp) [inline docs/conceptual/snippets/fssequences/snippet204.fs]: Let's add a line break before this comment.

File: docs/conceptual/seq.tail['t]-function-[fsharp].md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Version Information
+**F# Core Library Versions**
+
+Supported in: 4.0, Portable
+
+
+
+
+## See Also
```
Comment (cartermp) [inline docs/conceptual/seq.tail['t]-function-[fsharp].md]: Let's add a link to `Seq.head`, as this is the dual of `Seq.tail`.

File: docs/conceptual/seq.tail['t]-function-[fsharp].md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+## Exceptions
+
+|Exception|Condition|
+|----|----|
+|[ArgumentException](https://msdn.microsoft.com/library/system.argumentexception.aspx)|Thrown when the input sequence is empty.|
+|[ArgumentNullException](https://msdn.microsoft.com/library/system.argumentnullexception.aspx)|Thrown when the input sequence is null.|
+
+## Return Value
+
+The result sequence.
```
Comment (cartermp) [inline docs/conceptual/seq.tail['t]-function-[fsharp].md]: Let's be specific here.  The resulting sequence is a new sequence composed of the input sequence without its head.  Although that effectively copies the summary, saying "The result sequence" doesn't really add anything.  I'd rather we be slightly redundant.

We could cross-reference `Seq.head` here as well.

File: docs/conceptual/seq.tail['t]-function-[fsharp].md [hunk 5]
```diff
@@ -0,0 +1,76 @@
+---
+title: Seq.tail<'T> Function (F#)
+description: Seq.tail<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+---
+
+# Seq.tail<'T> Function (F#)
+
+Returns the sequence without the first element.
```
Comment (cartermp) [inline docs/conceptual/seq.tail['t]-function-[fsharp].md]: I'm not sure "the sequence" is crystal clear here.  Something more along the lines of "a new sequence composed of the input sequence without its first element" makes the behavior more clear.

**Comment by cartermp** [comment]
Left some minor feedback.  Good work overall :+1:

**Comment by cartermp** [comment]
LGTM.  Thanks @liboz!

**Comment by dend** [comment]
LGTM 💡

**Pull request merged**